packages/node_modules/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -362,33 +367,39 @@ skipInNode(describe)('plugin-meetings', () => {
             })
         ])));
 
-      it('alice shares the screen with highFrameRate', () => alice.meeting.getMediaStreams({
-        sendShare: true
-      }, {}, {highFrameRate: true})
-        .then((response) => Promise.all([
-          testUtils.delayedPromise(alice.meeting.updateShare({
-            sendShare: true,
-            receiveShare: true,
-            stream: response[1]
-          })),
-          testUtils.waitForEvents([{scope: alice.meeting, event: 'meeting:startedSharingLocal'}]),
-          testUtils.waitForEvents([{scope: bob.meeting.members, event: 'members:update'}])
-            .then((response) => {
-              console.log('SCREEN SHARE RESPONSE ', JSON.stringify(response));
-            }),
-          testUtils.waitForEvents([{scope: alice.meeting, event: 'media:ready'}])
-            .then((response) => {
-              console.log('MEDIA:READY event ', response[0].result);
-              assert.equal(response[0].result.type === 'localShare', true);
-            })
-        ]))
-        .then(() => {
-          assert.equal(alice.meeting.mediaProperties.shareTrack.getConstraints().height, 720);
-          assert.equal(alice.meeting.isSharing, true);
-          console.log('SCREEN SHARE PARTICIPANTS ', JSON.stringify(alice.meeting.locusInfo.participants));
+      it('alice shares the screen with highFrameRate', () => {
+        if (bowser.name.toLowerCase() === 'firefox') {
```

Comment:
[`isBrowser`](https://lancedikson.github.io/bowser/docs/Parser.html#isBrowser) seems to be a more robust way to do this.